dart/lib/src/scope.dart

```diff
@@ -398,17 +398,17 @@ class Scope {
       await clone.setExtra(extraKey, _extra[extraKey]);
     }
 
-    for (final breadcrumb in _breadcrumbs) {
+    for (final breadcrumb in List.from(_breadcrumbs)) {
       await clone.addBreadcrumb(breadcrumb);
     }
 
     for (final eventProcessor in _eventProcessors) {
       clone.addEventProcessor(eventProcessor);
     }
 
-    contexts.forEach((key, value) {
+    contexts.forEach((key, value) async {
```

Comment:
Shouldn't this also follow the same pattern as above to prevent a concurrent modification?

Reply:
Good question, opted to specifically fix the reported issue.